fog-kubevirt.gemspec

```diff
@@ -24,7 +24,6 @@ Gem::Specification.new do |spec|
   # spec.test_files   += %w[.rspec]
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency "bundler", "~> 1.3"
```

Comment:
Is this change related specifically to this PR ? Or is this the case

Reply:
There was new release of `bundler` and it version is different for ruby 2.3 and 2.4. Removing it allows to choose correct verison for specific ruby version which makes sure that travis is happy.